support/oidc-discovery-provider/config.go

```diff
@@ -308,6 +317,12 @@ func ParseConfig(hclConfig string) (_ *Config, err error) {
 			return nil, errors.New("the jwt_issuer url must contain a host")
 		}
 	}
+	if c.JWKSURI != "" {
+		jwksURI, err := url.Parse(c.JWKSURI)
+		if err != nil || jwksURI.Scheme == "" || jwksURI.Host == "" {
+			return nil, fmt.Errorf("the jwks_uri setting could not be parsed: %w", err)
+		}
+	}
```

Comment:
Could we also log something here if JWTIssuer is set, but not JWTKSURI? So users know that they are on a soon to be unsupported path. Maybe something like:

```
Warning: JWTIssuer configuration will also apply to the JWKS URI. This behaviour will be removed in 1.13.0.
```

Reply:
Great idea. Implemented.